vertx-core/src/test/java/io/vertx/tests/dns/NameResolverTest.java

```diff
@@ -866,7 +866,7 @@ public void testResolveAll() {
 
     List<String> expectedIPAddresses = Arrays.asList("127.0.0.2", "127.0.0.3");
 
-    dnsServer.addRecordsToStore( "fakeAddress.com", expectedIPAddresses.toArray(new String[0]));
+    //dnsServer.addRecordsToStore( "fakeAddress.com", expectedIPAddresses.toArray(new String[0]));
```

Comment:
I was not able to understand how name resolutions works so this whole test was not completed, little help would be awesome @vietj